moveit_planners/trajopt/package.xml

```diff
@@ -6,9 +6,7 @@
   <author email="omid.github@gmail.com">Omid Heidari</author>
 
   <maintainer email="omid.github@gmail.com">Omid Heidari</maintainer>
-  <maintainer email="moveit_releasers@googlegroups.com">MoveIt! Release Team</maintainer>
-
-  <license>BSD</license>
```

Comment:
This shouldn't be changed, please revert

Reply:
sorry about that